app/src/organisms/Desktop/ProtocolVisualization/StepDetailContainer.tsx

```diff
@@ -41,32 +65,25 @@ export function StepDetailContainer({
     rightMountPipetteName == null &&
     (leftMountPipetteName === 'p1000_96' || leftMountPipetteName === 'p200_96')
 
-  const { moduleEntities } = invariantContext
-  const moduleOnSlot = Object.entries(robotState.modules ?? {}).find(
-    ([_, module]) => module.slot === selectedSlot
-  )
-
   const leftPipetteId =
-    Object.entries(robotState.pipettes ?? {}).find(
+    Object.entries(pipettes ?? {}).find(
       ([_, pipette]) => pipette.mount === 'left'
     )?.[0] ?? null
   const rightPipetteId =
-    Object.entries(robotState.pipettes ?? {}).find(
+    Object.entries(pipettes ?? {}).find(
       ([_, pipette]) => pipette.mount === 'right'
     )?.[0] ?? null
 
   const isLeftPipetteActive =
-    selectedRunTimeCommand?.params != null &&
-    'pipetteId' in selectedRunTimeCommand.params &&
-    selectedRunTimeCommand.params.pipetteId === leftPipetteId &&
+    'pipetteId' in currentCommand.params &&
+    currentCommand.params.pipetteId === leftPipetteId &&
     leftPipetteId != null &&
-    robotState.pipettes[leftPipetteId].entityId != null
+    pipettes[leftPipetteId].entityId != null
   const isRightPipetteActive =
-    selectedRunTimeCommand?.params != null &&
-    'pipetteId' in selectedRunTimeCommand.params &&
-    selectedRunTimeCommand.params.pipetteId === rightPipetteId &&
+    'pipetteId' in currentCommand.params &&
+    currentCommand.params.pipetteId === rightPipetteId &&
     rightPipetteId != null &&
-    robotState.pipettes[rightPipetteId].entityId != null
+    pipettes[rightPipetteId].entityId != null
```

Comment:
If we don't need the IDs beyond getting if the pipettes are active, maybe we refactor this into a util?
```suggestion
  const getIsPipetteActive = (side: 'left' | 'right'): boolean => {
    const pipetteId =
      Object.entries(pipettes ?? {}).find(
        ([_, pipette]) => pipette.mount === side
      )?.[0] ?? null
    return (
      'pipetteId' in currentCommand.params &&
      currentCommand.params.pipetteId === pipetteId &&
      pipetteId != null &&
      pipettes[pipetteId].entityId != null
    )
  }

  const isLeftPipetteActive = getIsPipetteActive('left')
  const isRightPipetteActive = getIsPipetteActive('right')
```